pages/accreditation/[slug].vue

```diff
@@ -1,21 +1,50 @@
 <template>
   <div class="page">
-    <section class="bg-gradient-to-b from-sushi-50 to-sushi-100 md:pt-28 max-w-screen ">
-      <div class="page-fade-in contain flex flex-col justify-center items-center max-w-8xl pt-28 pb-20 md:pb-0 lg:pt-12 gap-10">
-        <h1>{{ accred?.data.hero_title || "Your money is building a greener future" }}</h1>
+    <section class="bg-gradient-to-b from-sushi-50 to-sushi-100 pt-28 max-w-screen">
+      <div class="page-fade-in relative contain pb-20 lg:pt-12 z-10">
+        <article class="relative flex flex-col items-center border rounded-2xl w-full max-w-7xl min-h-[504px] px-4 py-10 bg-white text-center gap-10">
+          <h1 class="text-3xl lg:text-5xl font-semibold text-gray-800">
+            <template v-if="accred?.data.hero_title">
```

Comment:
[template tag](https://vuejs.org/api/built-in-special-elements#template) can also be useful to avoid rendering unnecessary HTML tags like in this case where a div or other tag is not needed.